pkg/metrics/producers/scheduledcapacity/crontabs.go

```diff
@@ -0,0 +1,77 @@
+/*
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package scheduledcapacity
+
+import (
+	"fmt"
+	"github.com/awslabs/karpenter/pkg/apis/autoscaling/v1alpha1"
+	"github.com/robfig/cron/v3"
+	"strconv"
+	"strings"
+	"time"
+)
+
+type Crontab string
+
+func (tab *Crontab) String() string {
```

Comment:
Weird --  I wouldn't think it forces you to reimplement string. Maybe since string is an interface =/?. You might be able to skip this with 
```
type Crontab Struct {
  string // this is an inline type
}

```
I'm fine either way.

Reply:
Will check it out. It felt a little weird to me so maybe this will fix.